### ISSUE TITLE
`bin/rails credentials:edit` is ignoring the ENV['RAILS_MASTER_KEY']

### ISSUE THREAD

**Issue by stevenharman** [rails/rails#31917]
It seems that `bin/rails credentials:edit` ignores the Rails master key if it's set via `ENV` var rather than `config/master.key`.

### Steps to reproduce

1. Generate a new Rails app
1. `cd <new_app> && cat config/master.key` prints `8fe10c494a635b461cfe23bf24999433`
1. `bin/rails credentials:show` prints the contents, as expected.
1. `bin/rails credentials:edit` opens editor.
     On save and close of file we see `New credentials encrypted and saved.`
1. `rm config/master.key`
1. `bin/rails credentials:show` reports `Missing master key to decrypt credentials. See bin/rails credentials:help`
1. `RAILS_MASTER_KEY=8fe10c494a635b461cfe23bf24999433 bin/rails credentials:show` prints the contents, as expected.
1. `RAILS_MASTER_KEY=8fe10c494a635b461cfe23bf24999433 bin/rails credentials:edit` generates a new key:

```
Adding config/master.key to store the master encryption key: bbfeef5bedd12d45078725d8bf1a7e33

Save this in a password manager your team can access.

If you lose the key, no one, including you, can access anything encrypted with it.

      create  config/master.key
```

### Expected behavior
`bin/rails credentials:edit` should respect the `RAILS_MASTER_KEY` ENV Var.

### Actual behavior
Removing the `config/master.key` and invoking with an ENV var ignores the ENV Var:

```
$ RAILS_MASTER_KEY=8fe10c494a635b461cfe23bf24999433 bin/rails credentials:edit
Adding config/master.key to store the master encryption key: bbfeef5bedd12d45078725d8bf1a7e33

Save this in a password manager your team can access.

If you lose the key, no one, including you, can access anything encrypted with it.

      create  config/master.key

New credentials encrypted and saved.
```

### System configuration
**Rails version**: `5.2.0.rc1`

**Ruby version**: `2.5.0`

**Comment by y-yagi** [comment]
Thanks for your report. I will fix.

**Comment by sapnaSpaceo** [comment]
Hello this issue is fixed in rails 5.2 version or I will need rails 6 for this change? As I am facing same issue I have added the 'RAILS_MASTER_KEY' in my elastic beanstalk environment.But it is giving me this error ```ArgumentError: Missing `secret_key_base` for 'staging' environment, set this string with `rails credentials:edit` ```
